docs/source/technical-reference/index.rst

```diff
@@ -19,50 +19,61 @@ ScanCode Workbench Supported Platforms
-|  MacOS      | 10.9 Mavericks   |    x64  |     1      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
-|  MacOS      | 10.10 Yosemite   |    x64  |     2      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
-|  MacOS      | 10.11 El Capitan |    x64  |     2      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
-|  MacOS      | 10.12 Sierra     |    x64  |     2      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
-| Linux Deb   | Ubuntu 12.04     |    x64  |     1      | From Electron Docs: The prebuilt ia32   |
-|             |                  |         |            | (i686) and x64 (amd64) binaries of      |
-|             |                  |         |            | Electron are built on Ubuntu 12.04.     |
-+-------------+------------------+---------+------------+-----------------------------------------+
-| Linux Deb   | Ubuntu 14.xx     |    x64  |     2      | Verified to be able to run the prebuilt |
-|             |                  |         |            | binaries of Electron.                   |
-+-------------+------------------+---------+------------+-----------------------------------------+
-| Linux Deb   | Ubuntu 16.xx     |    x64  |     2      | Verified to be able to run the prebuilt |
-|             |                  |         |            | binaries of Electron.                   |
-+-------------+------------------+---------+------------+-----------------------------------------+
-|   Linux     |  Fedora 21       |    x64  |     2      | Verified to be able to run the prebuilt |
-|             |                  |         |            | binaries of Electron.                   |
-+-------------+------------------+---------+------------+-----------------------------------------+
-|   Linux     |  Debian 8        |    x64  |     2      | Verified to be able to run the prebuilt |
-|             |                  |         |            | binaries of Electron.                   |
-+-------------+------------------+---------+------------+-----------------------------------------+
-| Linux RH    |  CentOS 7.xx     |    x64  |     ?      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
-| Linux RH    |  RHEL 7.xx       |    x64  |     ?      |                                         |
-+-------------+------------------+---------+------------+-----------------------------------------+
++-------------+------------------+----------+------------+-----------------------------------------+
```

Comment:
Not sure if we've actually tested all these platforms, so let's remove this table and other details and just mention that we support linux/mac/macM1/windows platforms.

Reply:
oh, they were already there, i just added the latest versions of macOS, ubuntu, and windows11

but yeah, the latest workbench isn't tested on these specific distros, so it has only `linux/mac/macM1/windows platforms` makes sense

Reply:
Updated